pipelines/lib/util.groovy

```diff
@@ -2136,11 +2138,20 @@ def void runVerifyToSasquatch(Map p) {
           "$OUTPUT_COLLECTION" \
           --dataset "$dataset" \
           --url "$SASQUATCH_URL" \
-          --namespace "$SASQUATCH_NAMESPACE"
+          --namespace "$SASQUATCH_NAMESPACE" \
+          --extra "ci_id=$BUILD_ID" \
+          --extra "ci_url=$BUILD_URL" \
+          --extra "ci_name=$JOB_NAME" \
+          --extra "ci_refs=$JOB_REFS"
```

Comment:
Do keep in mind that since the avro schema is automatically built for the metric bundle it is happy to upgrade while it can, but will fail if you try to insert different types in the future.

Reply:
Sorry, what do you mean by "different types"? I thought adding fields was safe, and only removing them was forbidden. And the code is written so that `ci_refs` is always passed, even if it's the empty string.